src/test/java/de/tum/in/www1/artemis/notification/SingleUserNotificationServiceTest.java

```diff
@@ -241,17 +246,21 @@ void testNotifyUsersAboutAssessedExerciseSubmission() {
         Course testCourse = database.addCourseWithFileUploadExercise();
         Exercise testExercise = testCourse.getExercises().iterator().next();
 
-        User studentWithParticipationAndSubmissionAndResult = database.getUserByLogin(TEST_PREFIX + "student1");
-        User studentWithParticipationButWithoutSubmission = database.getUserByLogin(TEST_PREFIX + "student2");
+        User studentWithParticipationAndSubmissionAndAutomaticResult = database.getUserByLogin(TEST_PREFIX + "student1");
+        User studentWithParticipationAndSubmissionAndManualResult = database.getUserByLogin(TEST_PREFIX + "student2");
+        User studentWithParticipationButWithoutSubmission = database.getUserByLogin(TEST_PREFIX + "student3");
 
-        database.createParticipationSubmissionAndResult(testExercise.getId(), studentWithParticipationAndSubmissionAndResult, 10.0, 10.0, 50, true);
+        database.createParticipationSubmissionAndResult(testExercise.getId(), studentWithParticipationAndSubmissionAndAutomaticResult, 10.0, 10.0, 50, true);
+        Result manualResult = database.createParticipationSubmissionAndResult(testExercise.getId(), studentWithParticipationAndSubmissionAndManualResult, 10.0, 10.0, 50, true);
+        manualResult.setAssessmentType(AssessmentType.MANUAL);
+        resultRepository.save(manualResult);
         database.createAndSaveParticipationForExercise(testExercise, studentWithParticipationButWithoutSubmission.getLogin());
 
         testExercise = exerciseRepository.findAllExercisesByCourseId(testCourse.getId()).iterator().next();
 
         singleUserNotificationService.notifyUsersAboutAssessedExerciseSubmission(testExercise);
 
-        assertThat(notificationRepository.findAll()).as("Only one notification should have been created (for the user with a valid participation, submission, and result)")
+        assertThat(notificationRepository.findAll()).as("Only one notification should have been created (for the user with a valid participation, submission, and manual result)")
```

Comment:
Can you maybe also ensure that the notification actually got sent to only student3, and not student1?